### PR TITLE
add error messages, remove extra classes

### PR DESCRIPTION
**Pull request by Sild**

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/property/PropertyInsertTest.java [hunk 1]
```diff
@@ -375,7 +335,9 @@ public void testXXTimezoneUnsupported() throws Exception {
         JSONAssert.assertEquals("{\"error\":\"IllegalArgumentException: Invalid date format\"}", response.readEntity(String.class), true);
```
Comment (unrealwork) [inline src/test/java/com/axibase/tsd/api/method/property/PropertyInsertTest.java]: rewrite with extractErrorMessage method

File: src/test/java/com/axibase/tsd/api/Util.java [hunk 2]
```diff
@@ -11,9 +11,6 @@
 import java.text.SimpleDateFormat;
 import java.util.*;
 
-/**
- * @author Dmitry Korchagin.
- */
 public class Util {
 
     public static final Long MILLIS_IN_DAY = 1000 * 60 * 60 * 24L;
```
Comment (unrealwork) [inline src/test/java/com/axibase/tsd/api/Util.java]: TimeUnit.MILLISECONDS.convert(1, TimeUnit.DAYS);

Comment (Sild) [inline src/test/java/com/axibase/tsd/api/Util.java]: Then we should add the same method for all supported `TimeUnit`, which is too much and unnesessary

File: src/test/java/com/axibase/tsd/api/Util.java [hunk 3]
```diff
@@ -108,4 +105,12 @@ public static String transformDateToServerTimeZone(String date, int offsetMinute
...
         return ISOFormat(instance.getTime());
     }
+
+    public static StringBuilder copyChar(char c, int count) {
+        StringBuilder sb = new StringBuilder("");
+        for(int i = 0; i < count; i++) {
+            sb.append(c);
+        }
+        return sb;
+    }
```
Comment (rmakulov) [inline src/test/java/com/axibase/tsd/api/Util.java]: compare with generateStringFromChar

File: src/test/java/com/axibase/tsd/api/method/property/PropertyInsertTest.java [hunk 4]
```diff
@@ -388,7 +351,270 @@ public void testMillisecondsUnsupported() throws Exception {
...
+    @Test
+    public void testTagValueNull() throws Exception {
+        Property property = new Property("insert-property-t-3", "insert-property-e-3");
+        property.addTag("t1", null);
+        property.setDate("2016-06-09T09:50:00.000Z");
+
+        Response response = insertProperty(property);
+
+        assertEquals("Query should fail if tag contain only null values", BAD_REQUEST.getStatusCode(), response.getStatus());
+    }
```
Comment (rmakulov) [inline src/test/java/com/axibase/tsd/api/method/property/PropertyInsertTest.java]: please, add propertyExist checking

**Comment by rmakulov** [comment]
end review

**Comment by rmakulov** [comment]
lgtm

**Comment by unrealwork** [comment]
ok

**Pull request merged**